ragnarok_packets/src/position.rs

```diff
@@ -67,7 +128,9 @@ impl FromBytes for WorldPosition2 {
         let y1 = (coordinates[2] >> 4) | ((coordinates[1] & 0b111111) << 4);
         let x2 = (coordinates[3] >> 2) | ((coordinates[2] & 0b1111) << 6);
         let y2 = coordinates[4] | ((coordinates[3] & 0b11) << 8);
-        //let direction = ...
+        /* Not related with direction at all is bigger than 7, it seems to be always 8.
```

Comment:
If you do this, then you can add one `unknown` u8 field for the element, right? But have you checked other implementations for it's possible meaning?

Reply:
I checked it seems a shift from x1, y1, but it seems that until now the shift is always 0.

Reply:
It seems to be a height "level" information? We should at least read the field and call it "unknown" for now.

Reply:
Please also remove the comment at the "world_position_2" test case, since world_position_2 seems to have no orientation at all.

Reply:
Ok, I will do that.

Reply:
Don't know if it what I am thinking.